subprojects/docs/src/snippets/java/fixtures/tests/dependencyInsight.out

```diff
@@ -13,6 +14,8 @@ com.google.code.gson:gson:2.8.5 FAILED
                - Variant enforced-platform-compile provides com.google.code.gson:gson-derived-enforced-platform:2.8.5
                - Variant enforced-platform-runtime provides com.google.code.gson:gson-derived-enforced-platform:2.8.5
 
+          - Review the variant matching algorithm documentation at https://docs.gradle.org/0.0.0/userguide/variant_attributes.html#abm_algorithm
```

Comment:
Blah.  A couple of problems here:
1. Duplicate messages are ugly and weird
2. The trailing `:` on the first line gets considered part of the URL, I think.

Reply:
Fixed.